lib/grinder_files.dart

```diff
@@ -98,12 +100,210 @@ class FileSet {
 }
 
 /**
+ * A class to make it easier to manipulate file system entites. Once paths or
+ * entites are converted into `Path`s, they can be easily copied, deleted,
```

Comment:
Much of the non-path-manipulation functionality here is available on FileSystemEntity, but dart:io has no implementation of FileSystemEntity which can abstractly represent any type (file, directory, link).  Might be cool to have `Path` extend `FileSystemEntity`.  Just an idea.